packages/core/content-type-builder/server/src/controllers/index.ts

```diff
@@ -0,0 +1,16 @@
+import builder from './builder';
+import componentCategories from './component-categories';
+import components from './components';
+import contentTypes from './content-types';
+
+// to prevent breaking changes, we need to keep the hyphenated names for now
+// TODO: do we actually need to do this? Is it public? Investigate if it's safe to change the exports to use camelCase
```

Comment:
```suggestion
```

Reply:
I think we could actually rename them since this is admin, but I don't want to rock the boat during the ts migration. We can address this later.